packages/web-runtime/src/container/application/classic.ts

```diff
@@ -128,7 +127,8 @@ export const convertClassicApplication = async ({
     supportedLanguages
   })
 
-  await store.dispatch('registerApp', applicationScript.appInfo)
+  const appsStore = useAppsStore()
+  appsStore.registerApp(applicationScript.appInfo)
```

Comment:
Exciting - why was this async before?

Reply:
As discussed, because the signature of `store.dispatch` is async.